### PR TITLE
[IMP] rma_sale: shipping address portal default

### PR DESCRIPTION
**Pull request by chienandalu**
The original order shipping address will be selected by default

cc @Tecnativa TT26406

### CODE REVIEW

File: rma_sale/views/sale_portal_template.xml [hunk 1]
```diff
@@ -32,18 +32,17 @@
                             </div>
                             <t
                                 t-set="delivery_addresses"
-                                t-value="sale_order.partner_id.commercial_partner_id.mapped('child_ids').filtered(lambda x: x.type in ['contact', 'delivery'])"
+                                t-value="sale_order.partner_shipping_id | sale_order.partner_id.commercial_partner_id.mapped('child_ids').filtered(lambda x: x.type in ['contact', 'delivery'])"
```
Comment (pedrobaeza) [inline rma_sale/views/sale_portal_template.xml:35]: A sales order always have `partner_shipping_id`:
```suggestion
                                t-value="sale_order.partner_shipping_id"
```

File: rma_sale/views/sale_portal_template.xml [hunk 2]
```diff
@@ -32,18 +32,17 @@
...
                                 class="btn btn-primary btn-block mb8"
                                 type="button" data-toggle="collapse"
                                 data-target="#delivery_address_picker"
                                 aria-expanded="false"
-
                             ><i class="fa fa-truck" /> Choose a delivery address</button>
                             <div class="col-lg-12 collapse mt8" id="delivery_address_picker">
                                 <div data-toggle="buttons" class="row">
-                                    <label t-attf-class="card mr4 btn btn-light" t-foreach="delivery_addresses" t-as="address">
+                                    <label t-attf-class="card mr4 btn btn-light #{address == sale_order.partner_shipping_id and 'active' or ''}" t-foreach="delivery_addresses" t-as="address">
```
Comment (pedrobaeza) [inline rma_sale/views/sale_portal_template.xml:45]: Shouldn't this be the previous selected address?

```suggestion
                                    <label t-attf-class="card mr4 btn btn-light #{address == delivery_addresses and 'active' or ''}" t-foreach="delivery_addresses" t-as="address">
```

File: rma_sale/views/sale_portal_template.xml (hunk 1 above)
Comment (chienandalu) [reply rma_sale/views/sale_portal_template.xml:35]: This a bitwise operator not a boolean one

File: rma_sale/views/sale_portal_template.xml (hunk 2 above)
Comment (chienandalu) [reply rma_sale/views/sale_portal_template.xml:45]: No, for the reason exposed before

File: rma_sale/views/sale_portal_template.xml (hunk 1 above)
Comment (pedrobaeza) [reply rma_sale/views/sale_portal_template.xml:35]: And which is its purpose then?

Comment (chienandalu) [reply rma_sale/views/sale_portal_template.xml:35]: To have a recordset of the commercial entity children and the original shipping address (wich could be not on the other set) avoiding repeating the last

Comment (pedrobaeza) [reply rma_sale/views/sale_portal_template.xml:35]: OK

**Review by pedrobaeza — APPROVED** [review]

**Comment by chienandalu** [comment]
@ernestotejeda please, can you add your review?

**Review by ernestotejeda — APPROVED** [review]

**Pull request merged**